[docs] Modifying wording for selector examples (#4752)
DevExpress/testcafe: docs/articles/documentation/test-api/selecting-page-elements/examples-of-working-with-dom-elements.md

});
```

## Have a different use case?

> If none of the examples fit your requirements and you encounter any difficulties, let us know on StackOverflow.

Reviewer: ```suggestion
If none of the examples fit your requirements and you encounter difficulties, contact DevExpress on StackOverflow.
```

Author: > Contact DevExpress
may not be a suitable option in this case, as users cannot 'contact' us there, per se, as it is not really our service. StackOverflow can be used by anyone for any questions, the only option to connect with us is just let us know by mentioning our tag, which is also included.

Reviewer: ```suggestion
If none of the examples fit your requirements and you encounter difficulties, let us know on StackOverflow.
```
